content/pages/sprint/2023/10-hsinchu.rst

```diff
@@ -0,0 +1,189 @@
+========================================
+Scisprint 2023 October in Hsinchu
+========================================
+
+:date: 2023-10-03 17:00
+:url: sprint/2023/10-hsinchu
+:save_as: sprint/2023/10-hsinchu.html
+
+Date
+-----
+
+* Date: 21th October, Saturday, 2023
+* Time: 10:00 -- 17:00 (7 hours)
+
+|
+
+Agneda 
+-------
+
+* 10:00-10:30 Arrival and Seating 
+
+* 10:30-11:00 Project Introduction (Quick overview of each project)
+
+* 11:00-12:00 Coding Session 1
+
+* 12:00-13:20 Lunch Break (group update and discussion)
+
+* 13:30-14:20 Coding Session 2
+
+* 14:20-14:30 Group update (and discussion) session 2
+
+* 14:30-15:20 Coding Session 3
+
+* 15:20-15:30 Group update (and discussion) session 3
+
+* 15:30-16:30 Final Sprint
+
+* 16:30-17:00 Project Summary (Include but not limit to the progress on the day)
+
+* 17:00 Dismiss for dinner
+
+|
+
+About the Scisprint
+----------------------
+
+To join the sprint, please bring your laptop and `sign up <#sign-up>`__.  You are also 
+very welcome to bring your project. Please `contact us <#contact-us>`__ if you have any 
+questions.
+
+Scisprint, hosted by the sciwork community, is a monthly coding sprint. It would like to 
+facilitate discussions and exchanges among people in the fields of science, numerical 
+computation, and engineering. Participants, regardless of experience level, can gain valuable 
+development insights in this event.
+
+This event includes a `hacking session <#hacking-session>`__ and `career conversation <#career-conversation>`__.
+
+We would like to provide a supportive and friendly environment for all attendees to support more developers
+to join in the open-source communities. 
+
+Hacking Session
+------------------
+
+It aims to encourage collaboration and interaction among developers through project 
+participation. The projects cover various fields, including but not limited to science, 
+numerical computation, and engineering. You are also encouraged to share your own projects 
+in scisprint. Refer to `project list <#project-list>`__ below for more details.
+
+Project List
++++++++++++++
+
+modmesh
+^^^^^^^^
+
+Toolkit for solving partial differential equations and analysis of the results.
+
+- **Related Subjects:** Python, C++, PDE
+- **Project Link:** `Github <https://github.com/solvcon/modmesh>`__
+- **Project Contact:** Yung-Yu Chen (discord: @yyc#7718)
+
+|
+
+pyLiteracy
+^^^^^^^^^^^
+
+Mandarin LocativePeriodic-zai [在] and Repetitive-zai [再] parser based on syntactical 
+context. 
+
+- **Project Link:** `Github <https://github.com/Chenct-jonathan/Loc_zai_and_Rep_zai_parser>`__
+- **Project Contact:** Jonathan Chen (discord: @陳畯田), PeterWolf (discord: @PeterWolf#1422)
+
+|
+
+uTensor
```

Comment:
We are not sure whether uTensor will participate in the October sprint or not.
Just delete uTensor. If they will participate, we will add it back.

Reply:
OK. I have already made the modifications.